docs/high-availability.md

```diff
@@ -1,42 +1,68 @@
 # High availability
 
-In a basic setup with 3 nodes, Percona XtraDB Cluster will continue to function
-if you take any of the nodes down.
-At any point in time, you can shut down any node to perform maintenance
-or make configuration changes.
-Even in unplanned situations
-(like a node crashing or if it becomes unavailable over the network),
-the Percona XtraDB Cluster will continue to work
-and you’ll be able to run queries on working nodes.
-
-If there were changes to data while a node was down,
-there are two options that the node may use when it joins the cluster again:
-
-
-* **State Snapshot Transfer** (SST) is when all data is copied
-from one node to another.
-
-  SST is usually used when a new node joins the cluster
-  and receives all data from an existing node.
-  Percona XtraDB Cluster uses **xtrabackup** for SST.
-
-  SST using `xtrabackup` does not require the `READ LOCK` command
-  for the entire syncing process, only for syncing [.frm](glossary.md#frm) files (the same as with a regular backup).
-
-
-* Incremental State Transfer (IST) is when only incremental changes
-are copied from one node to another.
-
-  Even without locking your cluster in read-only state, SST may be intrusive
-  and disrupt normal operation of your services.
-  IST lets you avoid that.
-  If a node goes down for a short period of time,
-  it can fetch only those changes that happened while it was down.
-  IST is implemeted using a caching mechanism on nodes.
-  Each node contains a cache, ring-buffer (the size is configurable)
-  of last N changes, and the node is able to transfer part of this cache.
-  Obviously, IST can be done only if the amount of changes needed to transfer
-  is less than N. If it exceeds N, then the joining node has to perform SST.
+In a basic setup with three nodes, if you take any of the nodes down, Percona
+ XtraDB Cluster continues to function.
+At any point, you can shut down any node to perform maintenance
+or configuration changes.
+
+Even in unplanned situations (like a node crashing or if it becomes unavailable
+ over the network), you can run queries on working nodes. If a node is down and
+ the data has changed, there are two methods that the node may use when it
+ joins the cluster again:
+
+| Method | What happens | Description  |
+|---|---|--- |
+| SST  | The joiner node receives a full copy of the database state from
+ the donor node. | You initiate a Solid State Transfer (SST) when adding
+ a new node to a Galera cluster or when a node has fallen too far out of
+ sync. <br> Percona XtraDB Cluster uses `xtrabackup` for SST which does
+ not require the `READ LOCK` command for the entire process; only when syncing files. |
+| IST  | Only incremental changes are copied from one node to another. | This
+ operation can be used when a node is down for a short period.  |                                                                                                                
+
```

Comment:
The table is broken in preview https://percona-xtradb-cluster-docs-8-0-pr-138.onrender.com/high-availability.html